source/winUser.py

```diff
@@ -612,3 +616,55 @@ def SendInput(inputs):
 	n = len(inputs)
 	arr = (Input * n)(*inputs)
 	user32.SendInput(n, arr, sizeof(Input))
+
+# Windows Clipboard format constants
+CF_UNICODETEXT = 13
+
+@contextlib.contextmanager
+def openClipboard(hwndOwner=None):
+	"""
+	A context manager version of OpenClipboard from user32. 
+	Use as the expression of a 'with' statement, and CloseClipboard will automatically be called at the end. 
+	"""
+	if not windll.user32.OpenClipboard(hwndOwner):
+		raise RuntimeError("OpenClipboard failed")

```

Comment:
This ignores extended error info.

```suggestion
		raise ctypes.WinError()
```